### PR TITLE
Add Additional ECR Fields into the ECR to FHIR Conversion

### PR DESCRIPTION
**Pull request by BradySkylight**
# PULL REQUEST

## Summary
Additional fields have been requested by LAC that should be included in the Conversion/Message Parsing for ECR messages.  These are listed below:

-  Patient administrative sex - this is what we currently have as 'Gender'
-  Patient birth sex
-  Homelessness Status
-  Disability Status
-  Tribal Affiliation
-  Tribal Enrollment Status
-  Current Job Title
-  Present/Usual Industry
-  Present/Usual Occupation
-  Preferred Language
-  Pregnancy Status

The FHIR Converter Liquid Templates for ECR have been updated to include the Pregnancy Section and supporting Observations as well as additional modifications to include the extensions/observations as required by HL7 ECR FHIR Profiles.  The asset/test files have been updated to include the ECR CDA sections and data necessary to confirm the conversion process pulled these fields correctly as well as the end result of our FHIR Conversion process.  The resulting Bundle as been uploaded successfully to our FHIR Server in our Azure test environment.   A mapping spreadsheet is included in this PR.

## Related Issue
#693 
[#188](https://app.zenhub.com/workspaces/secret-6480bf2ee530095ab41ebbe9/issues/gh/cdcgov/phdi-azure/188)
#694 

## Additional Information
[Mapping Spreadsheet
](https://docs.google.com/spreadsheets/d/1puur_o_4Fz7gDFA2s8mRZDgFj4V4LhT_a9Cu3cIpE5k/edit#gid=0)

### CODE REVIEW

**Review by DanPaseltiner — APPROVED** [review]
LGTM. Thanks for all of this work and testing against the FHIR server @BradySkylight!

File: containers/fhir-converter/Templates/eCR/Resource/_ObservationHomeEnv.liquid [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        "category":
+        [
+            {
+                "coding":
+                [
+                    {
+                        {% if observationCategory -%}
+                        "system":"http://terminology.hl7.org/CodeSystem/observation-category",
+                        {% endif -%}
+                        {% if observationCategory -%}
```
Comment (emmastephenson) [inline containers/fhir-converter/Templates/eCR/Resource/_ObservationHomeEnv.liquid:29]: I'm sure this is just some weirdness with the liquid templating - but is it possible to have a single `if` block here?

```
if observationCategory
"system":"http://terminology.hl7.org/CodeSystem/observation-category",
"code":"{{ observationCategory }}",
endif
```

**Review by emmastephenson — APPROVED** [review]
Looks great Brady, thanks for all your diligence here! 🙇

File: containers/fhir-converter/Templates/eCR/Resource/_ObservationHomeEnv.liquid (hunk 1 above)
Comment (BradySkylight) [reply containers/fhir-converter/Templates/eCR/Resource/_ObservationHomeEnv.liquid:29]: Good catch.  Fixing now.

**Pull request merged**